infrastructure/dynamodbkutter.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
basic-dynamodb-table

does not say what it will save

Reply:
it is already changed in an earlier pull request of another branch